src/main/java/fi/otavanopisto/kuntaapi/server/integrations/management/ManagementMenuEntityUpdater.java

```diff
@@ -141,15 +141,15 @@ public void timeout(Timer timer) {
   }
 
   private void updateManagementMenu(DefaultApi api, OrganizationId organizationId, MenuId managementMenuId) {
-    ApiResponse<fi.otavanopisto.mwp.client.model.Menu> response = api.kuntaApiMenusIdGet(managementMenuId.getId());
+    ApiResponse<fi.metatavu.management.client.model.Menu> response = api.kuntaApiMenusIdGet(managementMenuId.getId());
     if (response.isOk()) {
       updateManagementMenu(api, organizationId, response.getResponse());
     } else {
       logger.warning(String.format("Finding organization %s menu failed on [%d] %s", managementMenuId.getId(), response.getStatus(), response.getMessage()));
     }
   }
 
-  private void updateManagementMenu(DefaultApi api, OrganizationId organizationId, fi.otavanopisto.mwp.client.model.Menu managementMenu) {
+  private void updateManagementMenu(DefaultApi api, OrganizationId organizationId, fi.metatavu.management.client.model.Menu managementMenu) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused private "updateManagementMenu" method. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AUnusedPrivateMethod)